ansible/roles/baselayout/tasks/partials/repo/centos7.yml

```diff
@@ -12,20 +12,12 @@
     state: present
 
 
-- name: centos7 | install scl for devtoolset-6
-  when: "arch != 'arm64' and arch != 'ppc64'"
+- name: centos7 | install scl for devtoolset-6, devtoolset-8
+  when: "arch != 'arm64'"
   yum:
     name: centos-release-scl
     state: present
 
-- name: centos7 | ppc64 | install sclo7 devtoolset-6
```

Comment:
In case you wonder -- this doesn't work anymore, the repo is 404, but the above works now for ppc64.